### PR TITLE
Run3-gex50X Match name/namespace and the geometry parameters in DDD/DD4Hep for EE

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Match name/namespace and the geometry parameters in DDD/DD4Hep for EE

#### PR validation:

Use the cfg's in SimG4Core/Configuration/test

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
@bsunanda Does this PR fixes any bugs, or is it also a preliminary step to understanding what is wrong?

**Comment by bsunanda** [comment]
@cvuosalo It again sets the name/namespace correctly for dd4hep - I could not test it on CMSSW with the other dd4hep build. If it is available and can resolve the issue of having the plugins in right order, I can debug that part. In this version I get identical (almost) results from ddd and dd4hep for EE. This is not an attempt for the EB

**Comment by civanch** [comment]
+1

mainly improved debug printout and only 2-lines fix, better to merge asap

File: Geometry/EcalCommonData/plugins/dd4hep/DDEcalEndcapAlgo.cc [hunk 1]
```diff
@@ -257,8 +287,8 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
 
     if (1 == iSCType) {  // standard SC in this block
       eeSCELog = ns.addVolumeNS(Volume(myns + ee.envName + std::to_string(iSCType), eeSCEnv, ns.material(ee.mat)));
-      eeSCALog = Volume(ee.alvName + std::to_string(iSCType), eeSCAlv, ns.material(ee.wallMat));
-      eeSCILog = Volume(ee.intName + std::to_string(iSCType), eeSCInt, ns.material(ee.mat));
+      eeSCALog = Volume(myns + ee.alvName + std::to_string(iSCType), eeSCAlv, ns.material(ee.wallMat));
```
Comment (silviodonato) [inline Geometry/EcalCommonData/plugins/dd4hep/DDEcalEndcapAlgo.cc:290]: @cvuosalo the main change is here

**Comment by silviodonato** [comment]
+1

**Pull request merged**